Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx

```diff
@@ -1342,6 +1342,7 @@ class TPCTimeSeries : public Task
                             << "nClITS=" << nClITS
                             << "chi2ITS=" << chi2ITS
                             << "chi2match_ITSTPC=" << chi2match_ITSTPC
+                            << "PID=" << trkOrig.getPID()
```

Comment:
Note that like that you are saving the PID as a class. If you want store just the integer, use `trkOrig.getPID().getID()`.

Reply:
Does it matter for the output file size?

@miranov25 do you prefer to store the class or just the ID?

Reply:
Storing integer will be indeed enough

Reply:
@matthias-kleiner : the difference is that to read it back you will need the class library loaded while the integer can be read with bare root.